packages/sources/coinmetrics/src/transport/lwba.ts

```diff
@@ -114,15 +107,8 @@ export const handleCryptoLwbaMessage = (
           quote,
         },
         response: {
-          result: res,
-          mid: res,
-          ask: Number(message.ask_price),
-          asksize: Number(message.ask_size),
-          bid: Number(message.bid_price),
-          bidsize: Number(message.bid_size),
-          spread: Number(message.spread),
+          result: null,
           data: {
-            result: res,
             // bid, mid, ask included here again.
             // Also kept outside data for backward compatability
```

Comment:
Now that we've checked we don't need to maintain backward compatibility, can we delete this comment? 

Would be confusing with the fields deleted

Reply:
good point, will remove here and the same comment in ncfx